### PR TITLE
build: update @ew-did-registry@0.8.1-alpha.996.0

### PR DESCRIPTION
**Pull request by nichonien**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-import": "2.25.4",
     "ganache-cli": "^6.12.2",
     "husky": "^4.3.8",
+    "keyv":"4.5.0",
```
Comment (JGiter) [inline package.json:71]: Was this added accidentially?

Comment (nichonien) [reply package.json:71]: This was added to resolve this error - https://github.com/energywebfoundation/ew-credentials/actions/runs/3369032554/jobs/5588238008#step:6:800

Comment (JGiter) [reply package.json:71]: Hmm, ok. There is also https://www.npmjs.com/package/@types/keyv

Comment (nichonien) [reply package.json:71]: I saw this as well, but it is deprecated so I thought not to use it.

**Review by JGiter — APPROVED** [review]

**Pull request merged**